### PR TITLE
Log available commands in case of `No such command` Exception in `execute_command` service

### PR DESCRIPTION
**Pull request by vlebourl**
This requires https://github.com/iMicknl/python-tahoma-api/pull/51

### CODE REVIEW

**Comment by iMicknl** [comment]
Do we really need this in our Home Assistant integration? We can easily pull the details for a user based on their details from the debug log, is there a need for them to do this themselves?

**Comment by vlebourl** [comment]
My use case was to find the exact command tu use to refresh the thermostat state by using the `execute_command` service. 
As a power user, I was able to get it from the api itself, but I would have liked to have a solution to dump the commands to the log.

File: custom_components/tahoma/__init__.py [hunk 1]
```diff
@@ -144,11 +145,25 @@ async def handle_execute_command(call):
...
-            "Home Assistant Service",
-        )
+        try:
+            await tahoma_coordinator.client.execute_command(
+                entity.unique_id,
+                Command(call.data.get("command"), call.data.get("args")),
+                "Home Assistant Service",
+            )
+        except Exception as e:
+            if e.args[0].startswith("No such command"):
```
Comment (tetienne) [inline custom_components/tahoma/__init__.py:155]: Would be nice to have a dedicated exception instead of parsing the message.

Comment (vlebourl) [reply custom_components/tahoma/__init__.py:155]: I know, but the client only throws Exception there

Comment (vlebourl) [reply custom_components/tahoma/__init__.py:155]: https://github.com/iMicknl/python-tahoma-api/blob/282ab59d56172d799ffeebb89fbfd07c7dd8fe57/pyhoma/client.py#L334

Comment (vlebourl) [reply custom_components/tahoma/__init__.py:155]: https://github.com/iMicknl/python-tahoma-api/pull/54

**Pull request closed without merging**